modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -646,6 +655,11 @@ const register = (editor: Editor) => {
     default: false
   });
 
+  registerOption('content_editable_state', {
```

Comment:
Maybe add a todo about this going away or just remove it now?

Reply:
We have a separate Jira to clean this up so don't want to touch it just yet.